roles/servicetelemetry/tasks/component_prometheus.yml

```diff
@@ -1,3 +1,76 @@
+- name: Construct oauth redirect reference
+  set_fact:
+    prom_oauth_redir_ref:
+      kind: OAuthRedirectReference
+      apiVersion: v1
+      reference:
```

Comment:
Oh that's different... not sure I've seen an object with `reference` before. Only seen `spec`.